src/Appwrite/Migration/Version/V19.php

```diff
@@ -640,6 +646,17 @@ private function getFunctionCommands(Document $function): string
         return $commands;
     }
 
+    private function migrateEnumAttributeSize(): void
+    {
+        foreach ($this->documentsIterator('attributes', [Query::equal('format', ['enum']), Query::lessThan('size', Database::LENGTH_KEY)]) as $attribute) {
```

Comment:
Let's line break the queries for readability:

```suggestion
        foreach ($this->documentsIterator('attributes', [
            Query::equal('format', ['enum']), 
            Query::lessThan('size', Database::LENGTH_KEY)
        ]) as $attribute) {
```